src/lexer.l

```diff
@@ -44,17 +44,13 @@ hspace   [ \t]
 vspace   [\n\r]
 space    {hspace}|{vspace}
 path     :(\\.|[_\-\./a-zA-Z0-9#$+\*])+
-builtin  arg[0-9]|args|cgroup|comm|cpid|numaid|cpu|ctx|curtask|elapsed|func|gid|pid|probe|rand|retval|sarg[0-9]|tid|uid|username|jiffies
-call     avg|buf|cat|cgroupid|clear|count|delete|exit|hist|join|kaddr|kptr|ksym|len|lhist|macaddr|max|min|ntop|override|print|printf|cgroup_path|reg|signal|stats|str|strerror|strftime|strncmp|strcontains|sum|system|time|uaddr|uptr|usym|zero|path|unwatch|bswap|skboutput|pton|debugf|has_key|percpu_kaddr
+builtin  arg[0-9]|args|cgroup|comm|cpid|numaid|cpu|ctx|curtask|elapsed|func|gid|pid|probe|rand|retval|sarg[0-9]|tid|uid|username|jiffies|nsecs|kstack|ustack
```

Comment:
This is now equivalent to IDENT for the purposes of parsing, will this be removed subsequently?

It's actually getting near the point where we could also remove the second parsing pass for C macro expansion, by automatically converting each defined C macro into a BPFtrace macro (IFF it is used). WDYT?

Reply:
> This is now equivalent to IDENT for the purposes of parsing, will this be removed subsequently?

Yes, working on removing these builtins in the lexer/parser as well but running into some weird edge cases.

> It's actually getting near the point where we could also remove the second parsing pass for C macro expansion, by automatically converting each defined C macro into a BPFtrace macro (IFF it is used). WDYT?

Sounds good to me. Let me look into that after this.